public/app/plugins/panel/timeseries/models.cue

```diff
@@ -32,7 +32,7 @@ Family: {
                     ui.PointsConfig
                     ui.AxisConfig
                     drawStyle?:    ui.DrawStyle
-                    gradientMode?: ui.GraphGradientMode
+                    gradientMode?: ui.GraphGradientMode | {label:string, value:string}
```

Comment:
this is introduced with v8 json testdata, sometimes it is a string, and other cases are structure, might worth to ask torkel.

Reply:
no this is not an object, just a string enum, https://github.com/grafana/grafana/blob/4afa74232f23afe1559d5df324e6c84ed37bbdf6/packages/grafana-ui/src/components/uPlot/config.ts#L234

Reply:
@torkelo this is to fix https://github.com/grafana/grafana/blob/main/devenv/dev-dashboards/datasource-testdata/new_features_in_v8.json#L1186, if the examples are wrong, should we clean them up instead?

Reply:
yes, that json is wrong

Reply:
the result of a bug